cypress/integration/org_events_page.spec.ts

```diff
@@ -1,21 +1,25 @@
 describe('/o/[orgId]/events', () => {
-    beforeEach(() => {
-        cy.intercept({
-            method: 'GET',
-            url: /\/api\/orgs\/1$/,
-        }, {
-            data: {
-                id: 1,
-                title: 'Mocked org',
-            },
+
+        it('contains name of organization', () => {
+            cy.intercept('GET', /\/api\/orgs\/1$/, { fixture: 'dummyOrg' });
+            cy.visit('/o/1/events');
+            cy.contains('Mocked org');
         });
-    });
 
-    it('contains name of organization', () => {
-        cy.visit('/o/1/events');
-        cy.contains('Mocked org');
-    });
-});
+        it('contains events', () => {
+            cy.visit('/o/1/events');
+            cy.get('[data-test="event"]').should('be.visible');
+        });
 
+        it('contains a placeholder if there are no events', () => {
+            cy.fixture('dummyEvents.json').then((dummyEvents)  => {
+                dummyEvents.data = [];
+                cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941\/actions$/, dummyEvents);
```

Comment:
Mutating `dummyEvents` here seems unnecessary. Instead, just ignore `dummyEvents` in this test, and use an inline object literal on line 17 like so:

```
cy.intercept('GET', /\/api\/orgs\/1\/campaigns\/941\/actions$/, { data: [] });